communication/communications.py

```diff
@@ -77,8 +78,30 @@ def _write(self, sig, opcode, arg, seqNo):
         Sends the code to the Arduino
         """
         checksum = self.create_checksum(arg, opcode)
-        opcode_string = "%d%d%03d%d\r" % (sig, opcode, arg, checksum)
+        opcode_string = "%d%d%03d%d%d\r" % (sig, opcode, arg, checksum, seqNo)
         self.to_robot(opcode_string)
+        
+        # Check twice if response received, if not - resend
+        success = False
+        while (!success):
+            for i in range(2):
```

Comment:
The intention is to be willing to wait for a response for up to 0.2 s,  but expect it after 0.1 s